### DISCUSSION TITLE
阅读时添加高亮和笔记无法显示，侧边栏显示注释也无法显示所作的注释，但增加条目笔记却可以提取注释形成笔记

### DISCUSSION THREAD

**Discussion by zhangyan000005** [windingwind/zotero-better-notes#1042] [Q&A]
![Uploading 111.png…]()

**Comment by zhangyan000005** [comment]
已安装最新版插件和软件，但使用异常，版本如下
7.0.0-beta.104+7e01a7d0e (64-bit)
1.1.4-beta.101

**Reply by windingwind** [reply]
图没上传

**Comment by zhangyan000005** [comment]
![111](https://github.com/user-attachments/assets/64e8ab9d-a479-41c6-8f00-dbd93b29424f)

**Reply by windingwind** [reply]
帮助-restart in troubleshooting mode。禁用所有插件后重试

**Comment by zhangyan000005** [comment]
禁用后重启已恢复，请问如何重新恢复我之前已经安装的插件呢，现在全部用不了了

**Comment by zhangyan000005** [comment]
现在恢复插件后还是和昨晚一样，无法标注注释，请问是插件还没适配最新版软件的原因吗

**Reply by windingwind** [reply]
请禁用其他所有插件，仅开启本插件，检查是否仍然存在问题。如果不存在，则是其他插件的问题

**Comment by zhangyan000005** [comment]
请问怎么仅开启本插件，我刚刚禁用后better notes也被禁用了，然后重新启动之后，其他插件也是全部重启了

**Reply by windingwind** [reply]
工具-插件-一个个禁用，重启

**Comment by zhangyan000005** [comment]
请问我应该怎么排除是哪个插件的问题呢

**Reply by manning120008** [reply]
经过排查是插件PDF Figure导致

**Comment by zhangyan000005** [comment]
已排除出是哪个插件的问题，非常感谢您的指导

**Reply by manning120008** [reply]
经过排查是插件PDF Figure导致

**Comment by deswin123** [comment]
找到了原因，我是PDF Figure插件的设置问题。打开标注即可
![Uploading image.png…]()

**Comment by zhangyan000005** [comment]
您的邮件我已收到，谢谢！
